templates/default/ckan_properties.ini.erb

```diff
@@ -222,7 +222,7 @@ ckan.feeds.author_link =
 
 ## Storage Settings
 
-ckan.max_resource_size = 200
+ckan.max_resource_size = 250
```

Comment:
any issues on the other layers above ckan?

Reply:
Apache doesn't have the LimitRequestBody directive, so it should be fine.